syntax="proto3";

message FullName{
    string firstName = 1;
    string secondName = 2;
    string fathersName = 3;
}